aws/table_aws_route53_resolver_endpoint.go

```diff
@@ -0,0 +1,315 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go/service/route53resolver"
+	"github.com/turbot/go-kit/types"
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+)
+
```

Comment:
```suggestion
```suggestion
//// TABLE DEFINITION
```
```